ffi/src/lib.rs

```diff
@@ -727,10 +732,10 @@ pub extern "C" fn get_response_type_string(response_type: ResponseType) -> *cons
 /// * `command_response_ptr` must be obtained from the `CommandResponse` returned in [`SuccessCallback`] from [`command`].
 /// * `command_response_ptr` must be valid until `free_command_response` is called.
 #[no_mangle]
-pub unsafe extern "C" fn free_command_response(command_response_ptr: *mut CommandResponse) {
+pub unsafe extern "C-unwind" fn free_command_response(command_response_ptr: *mut CommandResponse) {
     if !command_response_ptr.is_null() {
         let command_response = unsafe { Box::from_raw(command_response_ptr) };
-        free_command_response_elements(*command_response);
+        free_command_response_elements(unsafe { *command_response });
```

Comment:
is dereferncing a box an unsafe operation?

Reply:
`free_command_response_elements` should be marked as unsafe I guess

Reply:
Dereferencing a Box isn't unsafe. I don't know why this is here, probably a mistake.